code/modules/library/skill_learning/job_skillchips/chef.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/skillchip/job/chef
+	name = "B0RK-X3 skillchip"  // bork bork bork
+	desc = "This biochip faintly smells of garlic, which is odd for something that is normally wedged inside a user's brain. Consult a dietician before use."
+	skill_name = "Close Quarters Cooking"
+	skill_description = "A specialised form of self defence, developed by skilled sous-chef de cuisines. No man fights harder than a chef to defend his kitchen."
+	skill_icon = "utensils"
+	activate_message = "<span class='notice'>You can visualize how to defend your kitchen with martial arts.</span>"
+	deactivate_message = "<span class='notice'>You forget how to control your muscles to execute kicks, slams and restraints while in a kitchen environment.</span>"
+	var/datum/martial_art/cqc/under_siege/style
+
+/obj/item/skillchip/job/chef/Initialize()
+	. = ..()
+	style = new
```

Comment:
The format of how this skillchip works technically, is copied from how "krav maga gloves" currently work. If there's a better way to phrase it, or if there's some issues with Destroy, those gloves will need to be updated too.